libpdbg/dtb.c

```diff
@@ -107,6 +110,44 @@ static bool get_chipid(uint32_t *chip_id)
 	return true;
 }
 
+static char* get_p10_system_type()
+{
+	FILE* file = fopen("/proc/device-tree/model", "r");
+	if (file == NULL) {
+		pdbg_log(PDBG_ERROR, "Unable to read device-tree model file: %s", strerror(errno));
+		return "";
```

Comment:
Should we return a blank string or NULL as the return type is a char*. Which would be appropriate here?